.github/workflows/industrial_ci_action.yml

```diff
@@ -10,19 +10,50 @@ jobs:
     strategy:
       matrix:
         env:
-          - {ROS_DISTRO: foxy, ROS_REPO: main}
-          - {ROS_DISTRO: foxy, ROS_REPO: testing}
+          - {
+            CACHE_PREFIX: 'foxy-main-debug-ccov',
```

Comment:
The CACHE_PREFIX shouldn't be a user-defined value up here in matrix.env.
Rather, I suggest to move it into the env section further down below.

Reply:
It is per-matrix.  The reason I did this was to shorten the cache key lines and make them easier to read.